test/integration/controller/core/workload_controller_test.go

```diff
@@ -194,18 +197,28 @@ var _ = ginkgo.Describe("Workload controller", func() {
 				AdmittedWorkloads: 1,
 				UsedResources: kueue.UsedResources{
 					corev1.ResourceCPU: {
-						flavorOnDemand: {
+						flavorSpot: {
 							Total:    pointer.Quantity(resource.MustParse("2")),
 							Borrowed: nil,
 						},
 					},
 				},
-			}))
+				Conditions: []metav1.Condition{
+					{
+						Type:    kueue.ClusterQueueActive,
+						Status:  metav1.ConditionTrue,
+						Reason:  "Ready",
+						Message: "Can admit new workloads",
+					},
+				},
+			}, ignoreCQConditionTimestamps))
```

Comment:
just ignore the conditions as well. This is not where we should be testing that.

Reply:
Makes sense.

Reply:
Done.